samtools.wdl

```diff
@@ -452,6 +452,38 @@ task Merge {
     }
 }
 
+task Quickcheck {
+    input {
+        File inputBam
+
+        String dockerImage = "quay.io/biocontainers/samtools:1.16.1--h6899075_1"
+    }
+
+    command {
+        set -e
+        samtools quickcheck ~{inputBam}
+    }
+
+    output {
+        File outputBam = inputBam
+    }
+
+    runtime {
+        time_minutes: 5
+        docker: dockerImage
```

Comment:
can cpu and memory also be explicitly set? Otherwise it will fallback on the defaults, and these may not be adequate or way too high.

Reply:
oh, aye, the defaults. yes, set explicitly now, thank you for the reminder.